### PR TITLE
USSD bypass pregnancy flow

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr plz review

File: test/fixtures_registration.js [hunk 1]
```diff
@@ -1934,7 +1934,7 @@ module.exports = function() {
                 },
                 'url': "http://localhost:8002/api/v1/registrations/",
                 'data':  {
-                    "stage": "prebirth",
+                    //"stage": "prebirth",  // bypassing pregnancy flow
```
Comment (gsvr) [inline test/fixtures_registration.js]: We always need to submit the `stage`, it's a requirement in the backend.

**Comment by gsvr** [comment]
@NicolaasDieBaas Please review. If it makes sense what I've done, please apply the same to #94 .  The big reason for my changes is that we always need to submit the 'stage'.

**Comment by NicolaasDieBaas** [comment]
@gsvr looks good :+1: I'll apply the same to #94

**Pull request merged**